### PR TITLE
 Removed proxies in master makefile

### PR DESCRIPTION
**Pull request by karimull**
created needed directories for schroot and modify file permissions for conf file

### CODE REVIEW

File: osc-server-bom/bin/create-centos [hunk 1]
```diff
@@ -76,8 +77,13 @@ type=directory
 root-groups=sudo
 users=jenkins
 groups=vmidc
+preserve-environment=true
 EOF
 
+chmod 400 /etc/schroot/chroot.d/centos.conf
+
+mkdir -p /var/lib/jenkins/
```
Comment (arvindn05) [inline osc-server-bom/bin/create-centos]: move this to a jenkins specific file/run

File: osc-server-bom/bin/create-centos [hunk 2]
```diff
@@ -5,6 +5,7 @@ apt-get install schroot yum rpm
 #
 # Build the centos env for install
 #
+mkdir -p /var/www
```
Comment (arvindn05) [inline osc-server-bom/bin/create-centos]: remove www references

**Comment by arvindn05** [comment]
@karimull can i merge...looks good to me. 

fixes https://github.com/opensecuritycontroller/community/issues/87

**Comment by karimull** [comment]
still testing Arvind.., will let you know ASAP...

**Comment by karimull** [comment]
Can you please review new changes

**Comment by arvindn05** [comment]
@karimull let me know if i can merge the changes

**Pull request merged**